src/epics/revisions.js

```diff
@@ -19,11 +19,22 @@ const buildUrl = ( state ) => {
 	return url;
 };
 
-const fetchRevisions = ( action$, store ) => (
+export const hideLoader = ( action$ ) => (
+	action$
+		.ofType( 'PAGES_UPDATE' )
+		.flatMap( () => Observable.of( RevisionsActions.setStatusReady() ) )
+);
+
+export const showLoader = ( action$, store ) => (
 	action$
 		.filter( ( action ) => [ 'QUERY_UPDATE', 'QUERY_SET_VALUE', 'WIKIS_SET' ].includes( action.type ) )
-		// Ensure that all the necessary data is present.
 		.filter( () => !!store.getState().query.wiki && store.getState().wikis.size > 0 && store.getState().query.user.size > 0 )
+		.flatMap( () => Observable.of( RevisionsActions.setStatusFetching() ) )
+);
+
+export const fetchRevisions = ( action$, store ) => (
+	action$
+		.ofType( 'REVISIONS_FETCHING' )
```

Comment:
@dayllanmaza,

I would not rely on the status (what if it's getting another page?).. but we could create another action, like `REVISIONS_FETCH` that instruct it to fetch the revisions.

Reply:
Done